package.json

```diff
@@ -26,12 +26,11 @@
     "@vue/test-utils": "^2.0.0-rc.21",
     "howler": "^2.2.3",
     "mosha-vue-toastify": "^1.0.23",
-    "nuxt": "npm:nuxt3@3.0.0-rc.3-27545866.cd37a21",
+    "nuxt": "npm:nuxt3@^3.0.0-rc.3-27550969.a4a3cff",
```

Comment:
Is this a newer or older version of nuxt3?

Reply:
This is slightly newer, the last version had a build error that we had to fix just now. Yet another reason why we shouldn't be making updates this morning 😑